### PR TITLE
No more slow exe if $PWD is a slow file system 

### PR DESCRIPTION
**Pull request by poquirion**
Fix issue #838 

If the launcher is used, $PWD will be moved at the end of the python library search path

### CODE REVIEW

**Comment by poquirion** [comment]
We just fixed the bug in the laucher addtopath in the end!

File: scripts/sct_propseg.py [hunk 1]
```diff
@@ -10,11 +10,10 @@
 #
 # About the license: see the file LICENSE.TXT
 #########################################################################################
-
-from msct_parser import Parser
 import sys
-import sct_utils as sct
 import os
+from msct_parser import Parser
+import sct_utils as sct
```
Comment (zougloub) [inline scripts/sct_propseg.py]: Unrelated change, could you not stage it?

File: scripts/sct_utils.py [hunk 2]
```diff
@@ -12,16 +12,15 @@
 #
 # About the license: see the file LICENSE.TXT
 #########################################################################################
+import os
+import sys
```
Comment (zougloub) [inline scripts/sct_utils.py]: same here

Comment (poquirion) [reply scripts/sct_utils.py]: This one is a cleanup, `from sys import stdout` is not used anymore in the code.
